src/utils/changeElementVOVisibility.js

```diff
@@ -1,12 +1,44 @@
 /**
  * This source file is part of the Swift.org open source project
  *
- * Copyright (c) 2021 Apple Inc. and the Swift project authors
+ * Copyright (c) 2022 Apple Inc. and the Swift project authors
  * Licensed under Apache License v2.0 with Runtime Library Exception
  *
  * See https://swift.org/LICENSE.txt for license information
  * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
+import TabManager from 'docc-render/utils/TabManager';
+
+const PREFIX = 'data-original-';
```

Comment:
maybe something like this, so its more clear what the prefix is?

```suggestion
const OG_PREFIX = 'data-original-';
```